src/main/java/thaumicboots/item/boots/comet/ItemElectricCometBoots.java

```diff
@@ -122,8 +122,18 @@ public void onArmorTick(World world, EntityPlayer player, ItemStack itemStack) {
             } else {
                 player.jumpMovementFactor = 0.05F;
             }
-            if (player.fallDistance > 0.25F) {
-                player.fallDistance -= 0.25F;
+            if (player.fallDistance > 4.0F) {
+                player.fallDistance -= 0.50F;
+                if ((player.fallDistance > 3.5F)) {
```

Comment:
these double conditionals make no sense, they're always true if the outer one is true:
if x>4, then x-0.5 > 3.5, same in the other files

Reply:
It’s checking the fall damage height no? It’s checking if it’s greater than 4 (which is confirmed to work) then if it is, reducing it by a half block, and if it is still above 3.5 blocks it takes electric damage up to the limit

Reply:
At least that’s the intention

Reply:
Raven is right, if the first inequality is verified, then the second is always verified

Reply:
Yeahhh didn't see that until they mentioned it, thanks for fixing it